main.py

```diff
@@ -59,4 +61,6 @@ async def redirect():
     app_port = int(os.getenv("APP_PORT", "8000"))
     reload = environment == "dev"
 
-    uvicorn.run(app="main:app", host=app_host, port=app_port, reload=reload)
+    uvicorn.run(
+        app="main:app", host=app_host, port=app_port, reload=reload, loop="asyncio"
```

Comment:
We need to use the `asyncio` loop to enable the nested feature (which is necessary for llama parse).